### PR TITLE
abi-breaks

### PR DESCRIPTION
**Pull request by AlanGriffiths**
We've changed some ABIs since 2.17

Fixes: #3562

### CODE REVIEW

File: src/server/symbols.map [hunk 1]
```diff
@@ -392,7 +401,21 @@ global:
     mir::input::VirtualInputDevice::if_started_then*;
     mir::input::default_cursor_size*;
     mir::input::input_platform_from_graphics_module*;
+    mir::input::make_unique_context*;
     mir::input::probe_input_platforms*;
+    mir::input::receiver::XKBMapperRegistrar::XKBMapperRegistrar*;
```
Comment (mattkae) [inline src/server/symbols.map:406]: Huh? Why are these in the 2.17 stanza now?

Comment (mattkae) [reply src/server/symbols.map:406]: Is this the fault of the tool, or did you do this on  purpose?

**Review by mattkae — CHANGES_REQUESTED** [review]

File: src/server/symbols.map (hunk 1 above)
Comment (AlanGriffiths) [reply src/server/symbols.map:406]: Huh!? Everything should have been in the 2.18 stanza. Mo...

Comment (AlanGriffiths) [reply src/server/symbols.map:406]: Good catch. Fixed

**Review by mattkae — APPROVED** [review]
All symbols appear to be 2.18 now, so I am happy :)

**Pull request merged**